src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -36,7 +36,6 @@ public static function tearDownAfterClass(): void
     {
         TestHttpServer::stop(8067);
         TestHttpServer::stop(8077);
-        TestHttpServer::stop(8087);
```

Comment:
no need to stop this port, it can handle many test cases
(8067 and 8077 are special because they test situations where the webserver hangs)